.github/workflows/review.yaml

```diff
@@ -0,0 +1,22 @@
+name: 'Automagic Review'
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
### suggestion: Consider defining `on.pull_request` types more explicitly.  Have you considered adding more event types such as `edited` or `reopened`? This way, you ensure that the review process can adapt to changes after the initial creation.